projects/rucio-s3.yml

```
@@ -0,0 +1,46 @@
+---
+name: Rucio-S3-compatible access interface for analysis facilities
+postdate: 2023-05-07
+categories:
+  - Analysis tools
+  - Open science
+durations:
+  - 3 months
+skillset:
+  - Python
+  - C++
```

```suggestion
  - Docker
```

As the original proposal https://hepsoftwarefoundation.org/gsoc/2022/proposal_Rucio-S3.html mentions Docker but not C++ I'm going to swap these.